index.js

```diff
@@ -1 +1,130 @@
+const popup = document.querySelector(".popup");
 
+const inputFormAddTask = document.querySelectorAll(".form__input-task");
+const textareaFormAddTask = document.querySelector(".form__input-note");
+const openPopupButton = document.querySelector(".add-task__button");
+const closePopupButton = document.querySelector(".popup__close");
+const formSubmitButton = document.querySelector(".form__submit-btn");
+const taskList = document.querySelector(".todo__list");
+const taskBody = document.querySelector(".task-card__template");
+let taskRemove = document.querySelectorAll(".todo-item .todo-item__remove-btn");
+const form = document.querySelector('#addTask');
+
+function disabledSubmitButton() {
```

Comment:
в ситуация когде нет опасности потери контекста, лучше использовать стрелочные функции, меньше кода + аккуратнее выглядит:
```suggestion
const disabledSubmitButton = () => {
```